src/ontology/hp.Makefile

```diff
@@ -577,3 +577,33 @@ $(TMPDIR)/hp-%-merged.owl: hp-base.owl tmp/%.owl
 
 mappings: 
 	$(MAKE_FAST) ../mappings/hp-snomed.lexmatch.sssom.tsv
+
+##################
+### KGCL Diff ####
+##################
+
+.PHONY: kgcl-diff
+kgcl-diff: kgcl-diff-release-base
+
+.PHONY: kgcl-diff-md-release-base kgcl-diff-table-release-base kgcl-diff-txt-release-base kgcl-diff-yaml-release-base 
+kgcl-diff-md-release-base: reports/difference_release_base.md
+#kgcl-diff-table-release-base: reports/difference_release_base.tsv
```

Comment:
```suggestion
kgcl-diff-table-release-base: reports/difference_release_base.tsv
```

This command fails right now for me, with:

```
runoak -i simpleobo:tmp/hp-released.obo diff -X simpleobo:hp.obo \
-o reports/difference_release_base.tsv --output-type csv --statistics --group-by-property oio:hasOBONamespace
AttributeError: 'NoneType' object has no attribute 'subject'
make: *** [hp.Makefile:602: reports/difference_release_base.tsv] Error 1
```

Reply:
The latest form of the code seems to pass! This particular error matches the one I saw while working on https://github.com/monarch-initiative/rppr_stats/issues/1 and it is a bug in[ oaklib whose PR is waiting review ](https://github.com/INCATools/ontology-access-kit/pull/729)